run-cnf-suites.sh

```diff
@@ -118,17 +118,21 @@ if [ -n "$FOCUS" ]; then
 	fi
 fi
 
-if [ -n "$LABEL" ]; then
-    LABEL_STRING="-ginkgo.label-filter=${LABEL}"
+# Add the label "common" in case no labels have been provided. This will allow to filter out
+# some either non-official TCs or TCs not intended to run in CI (yet).
+# ToDo: remove when "-f" and "-s" flags have been deprecated.
+if [ -z "$LABEL" ]; then
+	LABEL="common"
 fi
+LABEL_STRING="-ginkgo.label-filter=${LABEL}"
 
 if [ -z "$FOCUS_STRING" ] && [ -z "$LABEL_STRING" ]; then
 	echo "No test focus (-f) or label (-l) was set, so only diagnostic functions will run.".
 fi
 
 
 
-cd ./cnf-certification-test && ./cnf-certification-test.test ${FOCUS_STRING} ${SKIP_STRING} ${LABEL_STRING:+"${LABEL_STRING}"} ${GINKGO_ARGS}
+cd ./cnf-certification-test && ./cnf-certification-test.test ${FOCUS_STRING} ${SKIP_STRING} ${LABEL_STRING} ${GINKGO_ARGS}
```

Comment:
DCI job is failing because of this changed line, see the error in execution.log file. For these labels provided:

```
-l access-control,(networking && !networking-dual-stack-service),lifecycle,observability,(platform-alteration && !platform-alteration-service-mesh-usage),operator,manageability,affiliated-certification-container-is-certified
```

Error displayed is:

```
[38;5;9mGinkgo detected configuration issues:[0m
[1m[38;5;9mSyntax Error Parsing Label Filter[0m
  access-control,[38;5;9m[1m[4m([0mnetworking
  Mismatched '(' - could not find matching ')'.
```